x11-packages/firefoxpwa/native_Cargo.toml.patch

```diff
@@ -0,0 +1,132 @@
+diff --git a/native/Cargo.toml b/native/Cargo.toml
+index 47315bf..8a04e83 100644
+--- a/native/Cargo.toml
++++ b/native/Cargo.toml
+
+@@ -70,14 +79,14 @@ features = [
+     "Win32_UI_WindowsAndMessaging",
+     "Win32_UI_Shell",
+     "Win32_UI_Shell_Common",
+-    "Win32_UI_Shell_PropertiesSystem"
++    "Win32_UI_Shell_PropertiesSystem",
+ ]
+
+-[target.'cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "openbsd", target_os = "netbsd"))'.dependencies]
++[target.'cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "openbsd", target_os = "netbsd", target_os = "android"))'.dependencies]
+ glob = "0.3.2"
+ phf = { version = "0.11.3", features = ["macros"] }
+
+-[target.'cfg(target_os = "linux")'.dependencies]
++[target.'cfg(any(target_os = "linux", target_os = "android"))'.dependencies]
+ blake3 = "1.5.5"
+ bzip2 = "0.5.1"
+ xz2 = "0.1.7"
+@@ -115,48 +121,74 @@ license-file = "packages/deb/copyright"
+ revision = ""
+ assets = [
+     # Executables
+-    ["target/release/firefoxpwa", "/usr/bin/firefoxpwa", "755"],
+-    ["target/release/firefoxpwa-connector", "/usr/libexec/firefoxpwa-connector", "755"],
++    [
++        "target/release/firefoxpwa",
++        "@TERMUX_PREFIX@/bin/firefoxpwa",
++        "755",
++    ],
++    [
++        "target/release/firefoxpwa-connector",
++        "@TERMUX_PREFIX@/libexec/firefoxpwa-connector",
++        "755",
++    ],
+
+     # Manifest
+-    ["manifests/linux.json", "/usr/lib/mozilla/native-messaging-hosts/firefoxpwa.json", "644"],
++    [
++        "manifests/termux.json",
++        "@TERMUX_HOME@/.mozilla/native-messaging-hosts/firefoxpwa.json",
```

Comment:
This should be moved to another place

Reply:
yes, i just notice it now. most of this patch for Cargo.toml is unnecesary as we don't use cargo-deb. i will resolve it shortly and see if there is another unnecesary patch.
thank you.